README.md

```diff
@@ -10,31 +10,36 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
 
-[Issue Example Here](https://github.com/arlene-perez/bot-app-test/issues/1)
-
-![Committing-Branch](/assets/Committing-Branch.png?raw=true)
-![Issue-Generated](/assets/Issue-Generated.png?raw=true)
-![Issue-Done](/assets/Issue-Done.png?raw=true)
-
-
-### ✅ ➕ Steps
+### ✅ ➕ Access Token Needed
 1) You'll need an Access Token from Github. Follow steps 1-9 from this [article](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) to generate your token. ⚠️ Make sure to SAVE the token in a document/editor file because it will only be shown to you once.
 
-2) Install [Node.js](https://nodejs.org/en/)
-
-3) Install [Hapi](https://hapijs.com/)
 
 ### 🕜 💻 Setup
 ```
 git clone https://github.com/hoodiehq/first-timers-only-bot.git
 cd first-timers-only-bot
-
+npm install
 ```
 
-### 🚦 🏁 How to Start the Server
+👀 When you are ready to use the bot, _make sure to change the label to **first-timers** in the instructions.md file._
+
+### 🚦 🏁 How to Start the Server Through Terminal
 
 In your terminal, type `TOKEN=<tokenhere> node server.js`. Make sure to substitute `<tokenhere>` for your access token from Github.
 
+### 🚦 🏁 How to Start the Server Through Glitch
+[Glitch](https://glitch.com/) is a platform where we hosted our first-timers-bot. It's so easy to use! Simply add your code and your bot will instantly be live!
+
 ### 😮 🙌 👀 🎉 Using the bot
+
+| 1) No need to fork the repo. Edit the file your change and commit message _directly_ on the repo where your Github App is installed on with. Make sure to click on "Create a new branch" at the bottom and begin the title of the branch with _first-timers-only_. | ![Committing-Branch](/assets/Committing-Branch.png?raw=true) |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------|
+| 2) Click on the issues tab and notice your issue was created with your change and commit message. The contributor would then follow the steps on the issue message.                                                                                                                               | ![Issue-Generated](/assets/Issue-Generated.png?raw=true)     |
```

Comment:
this currently renders the first row as headers and hence centers and bolds the text.
What you can do is to remove the `|------|-----|` line and put this on top

```
| | |
|-|-|
```

This will create a tiny empty row on top, but that’s the best we can do. Or use HTML code instead